src/include/test/unit_api_feature_vector_array.cc

```diff
@@ -46,22 +46,22 @@ TEST_CASE("api_feature_vector_array: test test", "[api_feature_vector_array]") {
 TEST_CASE("api: feature vector array open", "[api]") {
   tiledb::Context ctx;
 
-  auto a = FeatureVectorArray(ctx, sift_inputs_uri);
+  auto a = FeatureVectorArray(ctx, sift_inputs_uri.string());
```

Comment:
Since `FeatureVectorArray` has an overload for `std::filesystem::path`, we should use that instead of converting to a `string` here.  Similarly for other classes such as `FlatL2Index` (see comment above).